### PR TITLE
show command on multiple lines, increased readability

### PR DESCRIPTION
**Pull request by schmunk42**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,7 +111,13 @@ a 503.
 In order to be able to securize your virtual host, you have to create a file named as its equivalent VIRTUAL_HOST variable on directory
 /etc/nginx/htpasswd/$VIRTUAL_HOST
 
-    $ docker run -d -p 80:80 -p 443:443 -v /path/to/htpasswd:/etc/nginx/htpasswd -v /path/to/certs:/etc/nginx/certs -v /var/run/docker.sock:/tmp/docker.sock jwilder/nginx-proxy
+```
```
Comment (thaJeztah) [inline README.md]: While you're at it, perhaps make this

    ```bash

 for nicer code-highlighting?

**Comment by thaJeztah** [comment]
Nice!

File: README.md (hunk 1 above)
Comment (md5) [inline README.md]: Another option would be `console` highlighting.

Comment (thaJeztah) [inline README.md]: probably, yes. I'm just lazy and "bash" is easy to remember :)

Comment (schmunk42) [inline README.md]: hm, no changes for `bash` visible

bildschirmfoto 2015-04-09 um 22 36 52

But I'd drop the leading '$' for easier copy & pasting.

Comment (thaJeztah) [inline README.md]: Oh, that's a shame 😢

Not sure about dropping the `$`. I agree it's easier to copy/pasta but it's a bit of a convention to include it (at least in the docker docs)

**Comment by jwilder** [comment]
Thanks!

**Pull request merged**